website/www/site/content/en/documentation/sdks/java/testing/nexmark.md

```diff
@@ -632,30 +632,10 @@ There are dashboards for these runners (others to come):
 - spark
 - flink
 - direct runner
+- Dataflow
 
 Each dashboard contains:
 - graphs in batch mode
 - graphs in streaming mode
 - graphs for all the queries.
 
-### Performance dashboards links
-
-[Nexmark performance direct runner](https://apache-beam-testing.appspot.com/explore?dashboard=5084698770407424)
```

Comment:
Can we deep link similarly in the new dashboards?

Reply:
The equivalent dashboards don't exist yet. I filed https://issues.apache.org/jira/browse/BEAM-10643